components/AssetRow.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box, Flex } from "@chakra-ui/layout";
```

Comment:
I like it!. Thanks for making this its own component. Your attention to detail and organization is always appreciated.